CODEOWNERS

```diff
@@ -4,17 +4,12 @@ bigquery/   @jsimonweb
 bigtable/   @jsimonweb
 datastore/  @jsimonweb
 firestore/  @jsimonweb
-spanner/    @jsimonweb
 storage/    @jsimonweb
 applications/leaderboard/   @jsimonweb
 
-dialogflow/ @beccasaurus
-language/   @beccasaurus
-pubsub/     @beccasaurus
-speech/     @beccasaurus
-translate/  @beccasaurus
-video/      @beccasaurus
-vision/     @beccasaurus
+spanner/    @skuruppu
+
+kms/        @sethvargo

```

Comment:
Can you add me for `secretmanager/` too please?

Reply:
Will do, the more the better :)